### ISSUE TITLE
A11Y - @uifabric/experiments - pagination lose focus to the top of page in edge browser

### ISSUE THREAD

**Issue by achang0406** [microsoft/fluentui#10871]
### Environment Information

- **Package version(s)**: @uifabric/experiments 7.0.0
- **Browser and OS versions**: Microsoft Edge 44.18362.387.0 - Microsoft EdgeHTML 18.18362
- **Screen reader**: NVDA

### Describe the issue:

When trying to navigate a table with multiple pages using the pagination component in Microsoft Edge using only the keyboard..
Whenever one click on the pagination arrow to nav to the first or the last page of the table, the arrow will then be disabled because there are no more pages. However, this will cause the user to lose focus. The user will be brought to the top of the page when trying to tab again to the next or previous element.

### Please provide a reproduction of the issue in a codepen:
Repro:

1. navigate to the double right arrow on the pagination component and click "enter" with the keyboard
2. observe

#### Actual behavior:
User lose focus and is brought to the top of the page

#### Expected behavior:
User should focus on the next element on when click "tab" on the keyboard

### Documentation describing expected behavior
User should focus on the next or previous element on the page depending on if they click "tab" or "shift"+ "tab" on the keyboard.
User's focus on the page should not be brought to the top of the page.

@jgage176 
@caradong

**Comment by marygans** [comment]
Hi @achang0406 thank you for submitting this issue. Pagination component is not officially "supported" by the team since its in the [experiments package](https://github.com/OfficeDev/office-ui-fabric-react/tree/master/packages/experiments) and as a result, encouraged not to be used in product(See policy reference [here](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/experiments/README.md)).   @caradong is listed as the code owner for this component. 

I would suggest that you reach out to the original author and try to get it resolved that way! Also feel free to submit a PR :)

**Comment by jgage176** [comment]
I didn't realize I was supposed to comment on this. This still is a bug. Can it be reopened?

**Comment by ecraig12345** [comment]
I've reopened the issue, but you'll still need to work with the component owner to get it resolved (since this is not one of our official supported components).

**Comment by jgage176** [comment]
Sorry, I am still pretty new to GitHub and to collaborating on Office Fabric and don't know how to do things. How do I work with her except through this issue? I don't see any way to directly talk to her, Can I just tag her here? 

@caradong, we have stared using this component in multiple of our projects - I assumed that this was going to progress to production and there was no alternatives for having pagination besides writing our own, so I used it. We now have some accessibility issues and am hoping we can get them fixed and we can promote this into a first class component. It is possible I can help some this next month, if you can help me understand what we need to do to make this happen.

**Comment by ecraig12345** [comment]
Usually tagging people should work, but I tried chat messaging her just now (to make sure she saw the tag) and turns out she's on maternity leave with no indication of when she'll be back. I'll try and find out if anyone else from her team can help and get back to you.
